### DISCUSSION TITLE
Come posso creare due sensori Energia partendo da un unico sensore di potenza?

### DISCUSSION THREAD

**Discussion by mikysci** [indomus/forum#2270] [HUB: Home Assistant]
Ciao a tutti, sto iniziando ora con HA e quindi chiedo scusa se chiedo qualcosa che è già stato trattato (ho provato a cercare nel forum ma non ho trovato nulla). 

Ho un sensore che misura la corrente in entrata ed in uscita dalla Batteria del fotovoltaico (in Watt). Sensore unico, positivo se la corrente è in ingresso alla batteria, negativo se sto prelevando dalla batteria. Se creo il sensore come descritto nella guida Indomus "Calcolare il consumo di energia (in kWh) a partire dalla potenza istantanea (in Watt) con Home Assistant", mi da come risultato un sensore che riporta la "differenza" tra positivo e negativo. 
Come posso creare due sensori differenti (uno per l'immissione e uno per il prelievo)? 
Grazie mille
Michele

**Comment by dettofatto** [comment]
Hai più risolto? Grazie.

**Reply by Scialla** [reply]
dovete creare due sensori nuovi utilizzando il platform `filter` e le opzioni `lower_bound` e `upper_bound`

https://www.home-assistant.io/integrations/filter/

**Comment by dettofatto** [comment]
Grazie! Mi tocca un attimo studiare però...
